### ISSUE TITLE
Architecture: make virtual clusters use a virtual private DNS server

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#303]
Currently all virtual clusters use the DNS server that is managed by xCAT. This has the disadvantage that there is a DNS information leak, since containers and login nodes can see all DNS names for the different clusters (and also can see the DNS names for our infrastructure).

**Comment by hansthen** [comment]
I think this is part of the vlan implementation?

**Comment by abhimukh** [comment]
Yes, this is a part of the vlan implementation (r6_vlan). There is also a document describing the status.
